docs/develop/01_sdk/02_cookbook/01_dids/02_full_did_creation.md

```diff
@@ -23,10 +23,10 @@ If additional keys or service endpoints are to be specified, then they can be ba
 ## Upgrade a light DID to a full DID
 
 Another way to obtain a full DID is by upgrading a previously-created light DID.
-KILT supports this operation in a way that does not invalidate any credentials that have been issued to the light DID before being upgraded.
+KILT supports this operation in a way that does not invalidate any Credentials that have been issued to the light DID before being upgraded.
 
 The following code shows how to migrate a light DID to a full DID.
-Credentials, presentations, and verifications remain unchanged as adding support for DID migration does not affect the public API that the SDK exposes.
+Credentials, Presentations, and verifications remain unchanged as adding support for DID migration does not affect the public API that the SDK exposes.
```

Comment:
```suggestion
Credentials, Presentations, and Verifications remain unchanged as adding support for DID migration does not affect the public API that the SDK exposes.
```

Reply:
Verification is not a KILT entity, it does not need to be capitalized.